spec/imsc-hrm.html

```diff
@@ -399,28 +399,17 @@ <h2>Paint Regions</h2>
         <code>tts:extent="1920px 1080px"</code>, NSIZE(R<sub>i</sub>) ≈ 0.00603.
       </aside>
 
-      <p>NBG(R<sub>i</sub>) SHALL be the total number of <code>tts:backgroundColor</code> attributes associated with the given
-        region R<sub>i</sub> in the <a>intermediate synchronic document</a>. A <code>tts:backgroundColor</code> attribute is
-        associated with a region when it is explicitly specified (either as an attribute in the element, or by reference to a
-        declared style) in the following circumstances:</p>
-  
-        <ul>
-          <li>it is specified on the <code>region</code> layout element that defines the region; or</li>
-  
-          <li>it is specified on a <code>div</code>, <code>p</code>, <code>span</code> or <code>br</code> content element that is to
-          be flowed into the region for presentation in the <a>intermediate synchronic document</a> (see [[!ttml2]] for more details
-          on when a content element is followed into a region); or
-          </li>
-  
-          <li>it is specified on a <code>set</code> animation element that is to be applied to content elements that are to be flowed
-          into the region for presentation in the <a>intermediate synchronic document</a> (see [[!ttml2]] for more details on when a
-          <code>set</code> animation element is applied to content elements).
-          </li>
-        </ul>
-  
-        <p>Even if a specified <code>tts:backgroundColor</code> is the same as specified on the nearest ancestor content element or
-        animation element, specifying any <code>tts:backgroundColor</code> SHALL require an additional fill operation for all region
-        pixels.</p>
+      <p>NBG(R<sub>i</sub>) SHALL be the total number of elements within the tree rooted at region R<sub>i</sub> that satisfy the following criteria:</p>
+
+      <ul>
+        <li>the element is either a <code>region</code>, <code>body</code>, <code>div</code>, <code>p</code>, 
+           <code>span</code> or  <code>br</code>; and</li>
```

Comment:
Why not also resolve #3 by removing `br` from the list?

Reply:
See revised draft at https://github.com/w3c/imsc-hrm/pull/7/commits/1e997cb652dd71a7b22315ba43ffa05eeaead153